### ISSUE TITLE
Router zyxel vmg3625-t50b 

### ISSUE THREAD

**Issue by mohamedebrah** [mkst/zte-config-utility#98]
i just update my router with model VMG3625-50T(ABM6C0) , i cannot login my router any more , i already reset my router after updating software , i tried to login with login credentials printed on the back of my router but it is useles,s ,i tried with username admin and password 1234 and all common usernames and passwords but still the same problem ,can any one help me with this issue ?

this router is from my egyption internet service provider  , they refused to tell me admin password of the new software , i contact with zyxel company , it is usless as they said to me to contact with my isp
![1](https://github.com/user-attachments/assets/aa025e5b-e13c-4808-96d7-b57588af9e6c)
![2](https://github.com/user-attachments/assets/92c60b3d-b70e-4dfc-b91a-612995dfabe0)

**Comment by mkst** [comment]
Do you have a link to the firmware you used to update?

**Comment by mohamedebrah** [comment]
> Do you have a link to the firmware you used to update?

this is the link of firmware
https://www.mediafire.com/file/clw9b3p4qjfe2y3/V550ABPM6C0.zip/file

**Comment by mkst** [comment]
It's a shame that `admin:1234` hasn't worked as that's the username/password in the `/usr/etc/passwd` file for that firmware

```
admin:$1$$iC.dUsGpxNNJGeOm1dFio/:0:0:root:/:/bin/sh
```

Have you tried `admin:zyad1234` or `root`, `supervisor` or `admin` without any password?

**Comment by mohamedebrah** [comment]
> admin:zyad1234
here is the username is ( admin)  and password ( zyad1234)?


> `root`, `supervisor` or `admin`

as for those ,they all usernames without password , right

**Comment by mkst** [comment]
Yup

**Comment by mohamedebrah** [comment]
hello [mkst] , can you help me with that firmware also of model VMG3625-50t (ABPM2D0 ), i want the login credentials 
and this is the link of firmware https://www.mediafire.com/file/77a7wn1g32mjj08/V550ABPM2D0.rar/file

**Comment by maltahan9** [comment]
Did you find any solution?

**Comment by mido22w-art** [comment]
افتح الرابط واضغط على run
ثم اكتب السيريال كما هو على ظهر الراوتر ثم انتر
ويظهر الباسورد أسفل الصفحه
username : supervisr
https://www.onlinegdb.com/XR_spa_we?fbclid=IwY2xjawPFxwZleHRuA2FlbQIxMABicmlkETF5SEszN0dDZ1pCc0phYWg3c3J0YwZhcHBfaWQPNTE0NzcxNTY5MjI4MDYxAAEeaeP93ZBZ3sg-oUxQTiUMbDZfNKBFRVeD_FStbi-gOxRIs9FjOlZ3LoCRC28_aem_V0hVRDis9-6uxRrHFN_2Ew#

**Comment by hosamouda-19** [comment]
[ZyXEL_Keygen_GUI.zip](https://github.com/user-attachments/files/25692980/ZyXEL_Keygen_GUI.zip)
This is a small program I created myself using code to extract the username and password from the router's serial number. I hope everyone finds it useful.

<img width="464" height="322" alt="Image" src="https://github.com/user-attachments/assets/864b2fd2-6e41-4a49-8356-a64ce6e2be3f" />

**Comment by mido22w-art** [comment]
Thank you.
I need the firmware update file for vmg3625-t50b
Version v550abtl0b2r

**Comment by hosamouda-19** [comment]
This is the software version for the WE service provider device?
 you cannot install any external updates in the normal way. You must use the UART port from inside the router.

You can see the solution here.

https://forum.kitz.co.uk/index.php?topic=28198.msg476922;topicseen#new

**Comment by mido22w-art** [comment]
I contacted Zexel and they sent me an open-source file, but I don't know what to do with it. I'm not an expert in this field.

https://www.mediafire.com/file/vm2ozz7cucxc727/V550ABTL0b2r_release.tar/file?fbclid=IwY2xjawQRuJhleHRuA2FlbQIxMQBzcnRjBmFwcF9pZAwzNTA2ODU1MzE3MjgAAR63W2AgVrtj1xx2xXD2wPrdERHMTdZBTvRwQDDhixPi3QP2dl3txN5Q_cNe_w_aem_0zs5kxQORLyYYisGYIMOsQ

**Comment by hosamouda-19** [comment]
Did the software installation through the router's page work? If the installation method from the page didn't work, it means your device is locked from installing any files. Therefore, you should use the UART method, as it worked for me, and I installed the original Zyxel software from their website on my WE router.

**Comment by mido22w-art** [comment]
Could you please explain how to work with UART, mentioning the tools and software used?

**Comment by maltahan9** [comment]
Who are you

**Comment by mido22w-art** [comment]
What do you mean

**Comment by hosamouda-19** [comment]
راسلني علي الفيس بوك
https://www.facebook.com/share/1DLXqGhLv3/?mibextid=wwXIfr


> Could you please explain how to work with UART, mentioning the tools and software used?

**Comment by mido22w-art** [comment]
تم التواصل على الفيس بوك اسم الحساب mohamed sabry elmaghraby

**Comment by arishy** [comment]
I have the same router. Tried to use the USB port. Did not work. Called WE tech support!!!! They did not understand why I need to use the USB port.  This is the level of competence we are dealing with. When I called the support center I asked to talk to the manager. And that was his response.
 Any way; I need to get the original firmware and I do not know where I can find it. The manufacturer refused to deal with customized firmware made by ISP's

**Comment by hosamouda-19** [comment]
What problem did you encounter when applying the solution method via UART port?

**Comment by hosamouda-19** [comment]
It is not possible to revert to the original version from Zexel by updating the firmware through a browser because this method is blocked by the service provider, WE. The only way is to use TTL to UART port to access the bootloader using PuTTY, bypass the security, and then flash the software using TFTPD64.

<img width="585" height="1266" alt="Image" src="https://github.com/user-attachments/assets/c9984c15-b84f-4d13-9f0d-564b5887ac77" />

**Comment by arishy** [comment]
Great help. Just in case the "surgery" goes wrong Is there a way to get back to the current firmware. I hope I get a response from someone who actually did the above.

**Comment by hosamouda-19** [comment]
I and two other people have already done this and we didn't encounter any problems, so we didn't need to revert to the previous version. However, I believe you can revert to the original version by following the same steps after restarting the device and using the ATBT1 commands and those that follow.

**Comment by arishy** [comment]
This is very comforting to know that it is doable.  I appreciate your quick response.

**Comment by naderhelal-RouterUnlock** [comment]
اللي محتاج يفك تشفير الروتر دا ممكن يتواصل معايا مسنجر باسم نادر هلال أو من خلال صفحة الفيسبوك راوتر دكتور

If you need to unlock this router, you can contact me on Messenger under the name Nader Helal, or through my Facebook page Router Doctor.

https://www.facebook.com/share/g/17g8FSmKHq/
